### DISCUSSION TITLE
How can I display Netmiko send_command result in Django3 template

### DISCUSSION THREAD

**Discussion by schizo100** [ktbyers/netmiko#2145] [Q&A]
Hello Legends, 
How can I display Netmiko send_command result in Django3 template. 
I get None when a parse the output or I get the last result and Not all of it. I tried to write the output to a file but faced same issue.
with the function I cannot print the result / output because the function returns request, ...etc
I can only parse the output to a Jinja template. Is there a way to do it. 

Thank you. 

Here's the code:

def configure(request):
    
    all_devices = Devices.objects.all()
    output = ""

    if request.method == "POST":
        
            for i in all_devices:
                platform = "cisco_ios"
                ssh_connection = ConnectHandler(
                device_type=platform,
                ip=i.IpAdd,
                username=i.username,
                password=i.password,
                
                )
        
            output = ssh_connection.send_command("sh ip arp")
            

            with open('media/file', 'w') as f:

                
                output = f.write(line)
                    
                
                    
           ssh_connection.disconnect()
            

    

            

    
    context = {"all_devices": output}
    return render(request, "configure.html", context)

**Comment by ktbyers** [comment]
So what does `output` contain? Have you printed that out in Django "local" webserver (i.e. when running `python manage.py runserver`?

If that is empty or None, then you should debug your Netmiko problems completely outside of Django and get netmiko working properly.

**Reply by schizo100** [reply]
output variable contains the result from the send_command("") 
I can get the results in the terminal, However, I cannot parse the output to a Django template
This is NOT a Netmiko issue, However, Any help will be appreciated 
Here's the output on the terminal: 

February 10, 2021 - 22:21:29
Django version 3.1.5, using settings 'project.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.
Interface                  IP-Address      OK? Method Status                Protocol
FastEthernet0/0            192.168.15.184  YES NVRAM  up                    up      
Interface                  IP-Address      OK? Method Status                Protocol
FastEthernet0/0            192.168.15.185  YES NVRAM  up                    up      
Interface              IP-Address      OK? Method Status                Protocol
Ethernet0/0            192.168.15.186  YES NVRAM  up                    up      
Ethernet0/1            unassigned      YES unset  up                    up      
Ethernet0/2            unassigned      YES unset  up                    up      
Ethernet0/3            unassigned      YES unset  up                    up      
[10/Feb/2021 22:21:37] "POST /configure/ HTTP/1.1" 200 1474
